Tests/ML/visualizers/test_visualize_patches.py

```diff
@@ -37,6 +37,8 @@ def test_visualize_patch_sampling(test_output_dirs: OutputFolderForTests,
     shape = (10, 30, 30)
     foreground_classes = ["fg"]
     class_weights = equally_weighted_classes(foreground_classes)
+    # Initializes 'missing_labels_list' to 'False'
```

Comment:
Here and in other places: Comments that only repeat what the code does do not add much value, in my view.